client/components/Events/EventDateTime.tsx

```diff
@@ -72,13 +75,16 @@ export class EventDateTime extends React.PureComponent<IProps> {
                     date={start}
                     {...commonProps}
                 />
+                {!noEndTime && (
```

Comment:
the date part might be still relevant from the end timestamp, would it still display it?

Reply:
Yes, end date is also relevant to end time stamp. It would not display when no_end_time flag is true.
![image](https://user-images.githubusercontent.com/73937490/202096918-85237fde-7d8b-478c-b13a-be889e5119cd.png)
![image (1)](https://user-images.githubusercontent.com/73937490/202096939-c7bcdfc3-8f23-4fa9-8a7a-4ae1e731c6c5.png)

Reply:
the end date should be visible for multiday events even if `no_end_time` is set